src/Files.App/Utils/Storage/Helpers/FileThumbnailHelper.cs

```diff
@@ -45,14 +45,14 @@ public static async Task<byte[]> LoadIconFromStorageItemAsync(IStorageItem item,
 			return null;
 		}
 
-		public static async Task<byte[]> LoadIconFromPathAsync(string filePath, uint thumbnailSize, ThumbnailMode thumbnailMode, bool isFolder = false)
+		public static async Task<byte[]> LoadIconFromPathAsync(string filePath, uint thumbnailSize, ThumbnailMode thumbnailMode, ThumbnailOptions thumbnailOptions, bool isFolder = false)
```

Comment:
We can define `ThumbnailOptions.ResizeThumbnail` as the default value for `thumbnailOptions` to avoid adding the parameter to each method call.

Reply:
I'd rather it be explicit as different areas of the codebase use different options.

Reply:
Ok, then.